contracts/src/storage/AssetsStorage.sol

```diff
@@ -15,9 +14,13 @@ library AssetsStorage {
         uint128 assetHubReserveTransferFee;
         // Extra fee for registering a token, to discourage spamming (Ether)
         uint256 registerTokenFee;
+        // Token registry by token address
+        mapping(address token => TokenInfo) tokenRegistry;
+        // Token registry by tokenID
+        mapping(bytes32 tokenID => TokenInfo) tokenRegistryByID;
     }
 
-    bytes32 internal constant SLOT = keccak256("org.snowbridge.storage.assets");
+    bytes32 internal constant SLOT = keccak256("org.snowbridge.storage.assets.v2");
```

Comment:
Why is this called `SLOT`?

Reply:
Just by convention as each contract has a finite amount of storage space, which is divided into 32-byte slots.